### DISCUSSION TITLE
Add GeoMap option to fit data layers

### DISCUSSION THREAD

**Discussion by pbaumard** [grafana/grafana#44850] [Plugins]
Current [GeoMap](https://grafana.com/docs/grafana/latest/visualizations/geomap/) initial view options only allow a fixed center for the initial view.

The linked PR adds a _Fit data layers_ option using [OpenLayers fit](https://openlayers.org/en/latest/apidoc/module-ol_View-View.html#fit).

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
